### PR TITLE
Basic structure

### PR DESCRIPTION
**Pull request by Andyveloper**
# Requirements
- [x] Initialise React app.
- [x] Create a directory for your components.
- [x] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
## Books - the default view:
- [x] Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet).
- [x] Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- [x] Should have a form for adding a book (no functionality yet).
- [x] Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
## Categories:
- [x] Should display a button "Check status" only.
- [x] Styling is not required at this point.

### CODE REVIEW

File: src/components/Books/Book.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { CircularProgressbar } from 'react-circular-progressbar';
+
+const Book = () => {
```
Comment (EricMbouwe) [inline src/components/Books/Book.js]: - [ ] The `Book` component takes the book data as a prop in order to use them to display book data in the template. Let's fix that

File: src/components/Books/BookContainer.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    const newBook = {
+      id: uuidv4(),
+      title,
+      author,
+    };
+    setBooks([...books, newBook]);
+  };
+  return (
+    <div className="book-container">
+      <Book />
```
Comment (EricMbouwe) [inline src/components/Books/BookContainer.js]: - [ ] You should have a list of books return by the `Books` component. Currently there will always be only one book displayed. You can address that either by listing manually all your books or using the `map` function to iterate through your books state.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @Zeraltz, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters ok
- Good PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Zeraltz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**